docs/pages/application-access/getting-started.mdx

```diff
@@ -67,45 +90,66 @@ $ tctl tokens add \
     --app-uri=http://localhost:3000
 ```
 
-<Tabs>
-<TabItem label="Docker">
+Copy the output from this command and save the token in a separate file so it can be 
+passed to the Teleport Application Service.
 
-## Step 2/4. Start Grafana
+## Step 2/5. Start Grafana
 
-We've picked Grafana for this tutorial since it's very easy to run with zero
-configuration required. If you have another web application you'd like to
-expose, skip over to **Step 3**.
+You can start Grafana in a Docker container or Kubernetes cluster by running the 
+appropriate steps.
 
-Grafana can be launched in a Docker container with a single command:
+<Tabs>
+<TabItem label="Docker">
+
+You can start Grafana in a Docker container by running the following command:
 
 ```code
-$ docker run -d --name grafana -p 3000:3000 grafana/grafana
+$ docker run --detach --name grafana --publish 3000:3000 grafana/grafana
 ```
 
-Next, we need to edit `/etc/grafana/grafana.ini` in the container. Under
-`[server]`, uncomment the `domain` key and set the value to your Teleport
-cluster domain, (e.g., `teleport.example.com`). Since there is no text editor
-installed in the container, we can do this with `sed`:
+After running this command, edit the `/etc/grafana/grafana.ini` file in the container. 
+Under `[server]`, uncomment the `domain` key and set the value to your Teleport
+cluster domain, for example, `teleport.example.com`. Because the container doesn't 
+have a text editor installed, you can use the `sed` command to edit the file. 
+For example:
 
 ```code
-$ docker exec -u 0 grafana sed -i 's/;domain = localhost/domain = teleport.example.com/g' /etc/grafana/grafana.ini
+$ docker exec --user 0 grafana sed -i \
+  's/;domain = localhost/domain = teleport.example.com/g' \
+  /etc/grafana/grafana.ini
 ```
 
 Replace `teleport.example.com` with your Teleport cluster domain.
+</TabItem>
+<TabItem label="Kubernetes cluster">
 
-## Step 3/4. Install and configure Teleport
+If you install Grafana with the following `helm` instructions, it is available 
+at `http://example-grafana.example-grafana.svc.cluster.local` within the 
+Kubernetes cluster.
 
-(!docs/pages/includes/permission-warning.mdx!)
+```code
+$ helm repo add grafana https://grafana.github.io/helm-charts
+$ helm repo update
+$ helm install example-grafana grafana/grafana \
+    --create-namespace \
+    --namespace example-grafana
+```
+</TabItem>
+</Tabs>
+
+## Step 3/5. Install and configure Teleport
 
-### Configure and Start Teleport
+Install Teleport on the host that will run the Teleport Application Service.
```

Comment:
Why are we including this paragraph when there's a `TabItem` below that shows you how to run the Teleport Application Service in a Kubernetes pod?